crates/wysiwyg/src/composer_model/lists.rs

```diff
@@ -38,9 +38,18 @@ where
         // push_state_to_history is called if we can indent
         let (s, e) = self.safe_selection();
         let range = self.state.dom.find_range(s, e);
-        if !range.locations.is_empty() && self.can_indent(&range.locations) {
+        let locations: Vec<DomLocation> = range
+            .locations
+            .into_iter()
+            .filter(|l| l.relative_position() != Before)
```

Comment:
Is this filter correct ? Removing it doesn't break any test, but it fixes this: https://github.com/matrix-org/matrix-rich-text-editor/issues/506

Reply:
It was probably needed at the time, but the code for matching selections was changed at least 3 times, so it might no longer apply.